### PR TITLE
Dreamview:modify loop routing check threshold logic based 5.5

### PR DESCRIPTION
**Pull request by lj0919**

### CODE REVIEW

File: modules/dreamview/frontend/src/renderer/routing_editor.js [hunk 1]
```diff
@@ -140,4 +140,15 @@ export default class RoutingEditor {
     });
     WS.saveDefaultRouting(routingName, points);
   }
+
+  checkCycleRoutingAvailable(cycleRoutingPoints,carPosition,threshold) {
```
Comment (changsh726) [inline modules/dreamview/frontend/src/renderer/routing_editor.js:144]: (cycleRoutingPoints, carPosition, threshold)

File: modules/dreamview/frontend/src/components/DefaultRouting/CycleNumberInput.js [hunk 2]
```diff
@@ -34,8 +34,8 @@ export default class CycleNumberInput extends React.Component {
         alert('please input a valid cycle number');
       }
       else if (!routeEditingManager.checkCycleRoutingAvailable()) {
-        alert(`Please set the default routing reasonably,the distance from the start point
-          to the end point should not exceed ${routeEditingManager.defaultRoutingDistanceThreshold},
+        alert(`Please set the default routing reasonably,the distance from the car position
```
Comment (changsh726) [inline modules/dreamview/frontend/src/components/DefaultRouting/CycleNumberInput.js:37]: add a space after ,

File: modules/dreamview/frontend/src/renderer/index.js [hunk 3]
```diff
@@ -545,6 +545,11 @@ class Renderer {
         const names = intersects.map(intersect => intersect.object.name);
         return names;
     }
+
+    checkCycleRoutingAvailable(points,threshold) {
```
Comment (changsh726) [inline modules/dreamview/frontend/src/renderer/index.js:549]: (points, threshold)

File: modules/dreamview/frontend/src/renderer/routing_editor.js (hunk 1 above)
Comment (lj0919) [reply modules/dreamview/frontend/src/renderer/routing_editor.js:144]: thx

**Review by changsh726 — APPROVED** [review]

**Pull request merged**